spec/support/link_check_matchers.rb

```diff
@@ -10,4 +10,14 @@
       raise "Failed when checking URL: \"#{check_link}\""
     end
   end
+end
+
+RSpec::Matchers.define :be_a_working_link do
+  match do |check_link|
+    begin
+      good_statuses.include? Faraday.head(check_link).status
+    rescue => e
+      raise "Failed when checking \"#{check_link}\": #{e.message}"
```

Comment:
but... I moved the link checking into the `CmlessMarkdownTester` class, so the `be_a_working_link` matcher may not be used anymore.

Also... I'm not married to Faraday.